lib/php/lib/Transport/TSocket.php

```diff
@@ -251,8 +251,8 @@ public function open()
         }
 
         if (function_exists('socket_import_stream') && function_exists('socket_set_option')) {
-            $socket = socket_import_stream($this->handle_);
-            socket_set_option($socket, SOL_TCP, TCP_NODELAY, 1);
+            $socket = @socket_import_stream($this->handle_);
+            @socket_set_option($socket, SOL_TCP, TCP_NODELAY, 1);
         }
```

Comment:
I don't think this patch fixes anything. The [`@` operator in PHP][•], apparently, just silences errors.

However, it remedies the SSL warnings burden for (some of) the consumers of Thrift PHP library. 

Please at least add a comment saying ` // warnings silenced due to bug https://bugs.php.net/bug.php?id=70939` @sveneld 

[•]: https://stackoverflow.com/q/1032161/531179

Reply:
I understand that @ it does not fix trouble. I will add a comment